api-spec/api-spec.md

```diff
@@ -1,43 +1,45 @@
-## STAC API Specification
+# STAC API Specification
+
+The STAC API is intended to be a superset of the WFS 3.0 API specification.
```

Comment:
I think this should be just WFS 3, at least we call it like this in all the other documents.

Reply:
Fixed.